architecture/additional_concepts/authentication.adoc

```diff
@@ -674,3 +674,13 @@ responses from `/oauth/authorize`, and how they should be handled:
 |401    | `WWW-Authenticate` header missing                                                                                                                | No challenge authentication is possible. Fail and show response body (which might contain links or details on alternate methods to obtain an OAuth token)
 |Other  | Other                                                                                                                                            | Fail, optionally surfacing response body to the user
 |===
+
+[[authentication-prometheus-system-metrics]]
+=== Authentication metrics for Prometheus
+
+OpenShift captures the following Prometheus system metrics duing authentication attempts:
+* `openshift_auth_basic_password_count` counts the number of `oc login` username and password attempts
+* `openshift_auth_basic_password_count_result` counts the number of `oc login` username and password attempts by result (success or error)
+* `openshift_auth_form_password_count` counts the number of web console login attempts
+* `openshift_auth_form_password_count_result` counts the number of web console login attempts by result (success or error)
+* `openshift_auth_password_total` counts the total number of `oc login` and web console login attempts
```

Comment:
@mrogers950 Just a few comments from me. Otherwise, looks good! Thanks for this!

@bmcelvee I think you're helping to write up some content regarding Prometheus. Is this related to that? If so, do you have any opinions on the placement of this content? Thanks!